cloudformation/bulkExport.yaml

```diff
@@ -42,6 +42,33 @@ Resources:
 
   GlueScriptsBucket:
     Type: AWS::S3::Bucket
+    Properties:
+      BucketEncryption:
+        ServerSideEncryptionConfiguration:

```

Comment:
Do we want to use the s3 KMS key?

Reply:
default encryption is sufficient for a bucket that only has a copy of the python scripts from github. Also, using KMS with Glue has some added complexity in how the Glue Job permissions are managed.